Makefile

```
@@ -1,16 +1,19 @@
 # Makefile for gemini-cli
+# Enhanced with Nix integration
 
-.PHONY: help install build build-sandbox build-all test lint format preflight clean start debug release run-npx create-alias
+.PHONY: help install build build-sandbox build-all test lint format preflight clean start debug release run-npx create-alias help-nix clean-nix nix-build nix-test nix-shell nix-run inspect-bundle check-nix-deps nix-flake-update lint-nix nix-flake-show test-all dev-setup
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)